contracts/Uniswap/UniswapAnchoredView.sol

```diff
@@ -10,7 +10,7 @@ import "../Chainlink/AggregatorValidatorInterface.sol";
 
 struct Observation {
     uint timestamp;
-    uint acc;
+    int24 acc;
```

Comment:
Reason for this change?

Reply:
I've just removed it, since we are no longer recording observations/keeping our own TWAP. This was leftover from the code on Kovan - the accumulative ticks were being recorded as observations, instead of prices.